x-pack/legacy/plugins/epm/package.json

```diff
@@ -5,7 +5,7 @@
     "private": true,
     "license": "Elastic-License",
     "dependencies": {
+      "@elastic/package-registry": "ssh://git@github.com:jfsiii/package-registry.git#ts-types",
```

Comment:
Consuming from my PR/fork right now. This can become `github.com:elastic/package-registry` in the short term, then `@elastic/package-registry` eventually.

Reply:
Should this be https:// instead of ssh:// ? I guess no user permissions are needed.

Reply:
Yep, you're right. Updated in 80a5a26.

Will update to `elastic/package-registry` before merging this.